package.json

```diff
@@ -248,7 +248,7 @@
 		"design-system:dev": "echo \"Please use storybook:dev instead.\"",
 		"playground:build": "npm run storybook:build",
 		"playground:dev": "echo \"Please use storybook:dev instead.\"",
-		"env": "wp-scripts env",
+		"env": "packages/env/bin/wp-env",
```

Comment:
This suggests that wp-env and wp-script env have a similar external API, which I don't know if they have. An alternative to this would be to remove the npm `env` script

Reply:
cc @epiqueras @noahtallen

Reply:
Other than `start`/`stop`, the API is totally different :) A lot of the underlying stuff has changed too, so it's probably worth looking at the docs (via the deprecation message) to see what has changed